src/posts/spotlights/5-european-cities.md

```diff
@@ -0,0 +1,147 @@
+
+**What to do**
+
+You can start your visit from Verona Arena, one of the best preserved roman amphitheaters. The 2000 years old building is impressive and regularly hosts concerts and opera performances in front of 15,000 people.
+
+<figure>
+  <img src="/img/italy/verona-arena.jpg" alt="">
+  <figcaption>The 2000 years old Verona Arena.</figcaption>
+</figure>
+
+The rest of Verona has its roots in medieval times. Walking around the old town, you will find a lot of churches, palaces as well as small houses built in the specific italian architectural style.
+
+Verona was the imaginary setting for Shakespeare's "Romeo and Juliete" and "The Two Gentleman of Verona". And even though it seems Shakespeare never visited their city, the people here take advantage of the added publicity. There's a Juliete house in the city center, which has hordes of visitors throughout the year.
+
+However, if you are more interested in authentic sites, the church of [Sant'Anastasia](https://en.wikipedia.org/wiki/Sant%27Anastasia,_Verona) has one of the most impressive interior designs we've ever seen. Finally, if you want to do some cardio, the 84m tall Torre dei Lamberti awaits you in the central square with unparalleled views of the city.
+
+## Utrecht
+
+<figure>
+  <img src="/img/the-netherlands/utrecht.jpg" alt="">
+  <figcaption>Utrecht central area with the Domtoren in sight.</figcaption>
+</figure>
+
+**Why bother**
+
+For the last entry we are coming back to Western Europe. We've been to the Netherlands twice and visited a lot of cities, big and small. But Utrecht comes high of our list of a number of reasons.
+
+It is located in the central part of the country, making it the best place to stay while visiting the whole country. It also has a sizeable students population as a big university center. Finally, its city center is almost entirely car free, something you don't see in Amsterdam or in other cities in the area.
+
+**What to do**
+
+You can start your morning with a stroll along the Oudegracht canal, lined with old houses and shops. All roads in Utrecht lead to Domtoren, or Dom Tower, the symbol of the city. This is the tallest belfry in the Netherlands and used to be part of [St. Martin's Cathedral](https://en.wikipedia.org/wiki/St._Martin%27s_Cathedral,_Utrecht) until its nave collapsed in the 17th century. Since then, the church and the tower are separated by the Domplein square.
+
+<figure>
+  <img src="/img/the-netherlands/utrecht-canal.jpg" alt="">
+  <figcaption>The Oudegracht canal packed with bars, restaurants and places to relax.</figcaption>
+</figure>
+
+Utrecht hosts the biggest university in the Netherlands, so you can expect the city to be pretty packed throughout the year and very active during the evenings. It is also strategically positioned in the heart of the country, so a lot of the other major cities of the country are 20-30 minutes away by train.
```

Comment:
The fact that the city is located in the middle of the country is mentioned twice.